anima-jws-voucher.mkd

```diff
@@ -50,52 +50,45 @@ normative:
+"A Voucher Artifact for Bootstrapping Protocols" {{I-D.ietf-anima-rfc8366bis}} defines a YANG-based data structure
+used in "Bootstrapping Remote Secure Key Infrastructure" (BRSKI) {{!RFC8995}} and "Secure Zero Touch Provisioning" (SZTP) {{?RFC8572}}
+to transfer ownership of a device from a manufacturer to a new owner (customer or operational domain).
 That document provides a serialization of the voucher data to JSON {{RFC8259}} with cryptographic signing according to the Cryptographic Message Syntax (CMS) {{?RFC5652}}.
-The resulting voucher artifact has the media type "application/voucher-cms+json".
+That resulting voucher artifact has the media type `application/voucher-cms+json`.
 
-This document provides cryptographic signing of the JSON Voucher Data in form of JSON Web Signature (JWS) {{RFC7515}}
-and the media type "application/voucher-jws+json".
+This document provides cryptographic signing of voucher data in form of JSON Web Signature (JWS) {{RFC7515}} and the media type `application/voucher-jws+json` to identify the voucher format.
 The encoding specified in this document is used by {{?I-D.ietf-anima-brski-prm}}
 and may be more handy for use cases already using Javascript Object Signing and Encryption (JOSE).
-This document should be considered as enhancement of {{I-D.ietf-anima-rfc8366bis}},
-as it provides a new voucher form with media type "application/voucher-jws+json" and the related serialization.
-It does not extend the YANG definition of {{I-D.ietf-anima-rfc8366bis}}.
 
-Another example of an enhancement of {{I-D.ietf-anima-rfc8366bis}} is {{?I-D.ietf-anima-constrained-voucher}}.
-It provides the serialization of voucher data to CBOR {{?RFC8949}},
-with the signature in form of COSE {{?RFC8812}} and the media type "application/voucher-cose+cbor".
+This document should be considered as enhancement of {{I-D.ietf-anima-rfc8366bis}}, as it provides a new voucher format.
+It is similar to {{?I-D.ietf-anima-constrained-voucher}}, which provides cryptographic signing according COSE {{?RFC8812}} and the media type `application/voucher-cose+cbor`.
+These documents do not change nor extend the YANG definitions of {{I-D.ietf-anima-rfc8366bis}}.
 
-With the availability of different encoded vouchers, it is up to an industry specific application statement to indicate/decide which voucher signature format is to be used.
-There is no provision across the different voucher signature formats that a receiver could safely recognize which format it uses unless additional context is provided.
-For example, {{BRSKI}} provides this context via the media type for the voucher artifact.
-This document utilizes the optional "typ" (Type) Header Parameter of JWS {{RFC7515}} to provide information about the signed object.
+With the availability of different voucher formats, it is up to an industry-specific application statement to decide which format is to be used.
+The associated media types are used to distinguish different voucher formats.
```

Comment:
The previous text seemed to invent a problem that does not exist. Media types are exactly how this is solved.